iohub/ngff.py

```diff
@@ -1564,6 +1566,61 @@ def positions(self) -> Generator[tuple[str, Position], None, None]:
         for _, well in self.wells():
             for _, position in well.positions():
                 yield position.zgroup.path, position
+    
+    def rename_wells(self, well_names_csv: str):
+        """Modifies well names (and relevant zattrs) according to old and new paths specified in csv file. 
+
+        CSV input should be in the format:
+
+        oldrow/oldcolumn, newrow/newcolumn
+        
+        For example:
+
+        0/0, 0/A
+        0/1, 0/B
+        0/2, 0/C
+
+        rename_wells will adjust well paths from 0/0 --> 0/A, 0/1  -> 0/B
+
+        """
+        
+        names = []
+
+        with open(well_names_csv, mode='r', newline='', encoding='utf-8') as csvfile:
+            csvreader = csv.reader(csvfile)
+            for row in csvreader:
+                names.append([row[0], row[1]]) 
```

Comment:
Instead of taking a csv into this function, the CLI can handle the csv. So this block can move up to the CLI level. 

I'm imagining that the new signature of this function will let you  modify a single well via python, and the CLI level will let you rename many wells with a csv.

Reply:
It'll be worthwhile learning how to use click to build a CLI because you'll need to do the same for zebrapro.